src/components/SelectionList/Search/TaskListItemRow.tsx

```diff
@@ -0,0 +1,260 @@
+import React from 'react';
+import type {StyleProp, ViewStyle} from 'react-native';
+import {View} from 'react-native';
+import Avatar from '@components/Avatar';
+import Badge from '@components/Badge';
+import Button from '@components/Button';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import type {TaskListItemType} from '@components/SelectionList/types';
+import TextWithTooltip from '@components/TextWithTooltip';
+import useLocalize from '@hooks/useLocalize';
+import useNetwork from '@hooks/useNetwork';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {callFunctionIfActionIsAllowed} from '@libs/actions/Session';
+import {completeTask} from '@libs/actions/Task';
+import DateUtils from '@libs/DateUtils';
+import Parser from '@libs/Parser';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+import ReportInfoCell from './ReportInfoCell';
+import UserInfoCell from './UserInfoCell';
+
+type TaskListItemRowProps = {
+    item: TaskListItemType;
+    showTooltip: boolean;
+    containerStyle?: StyleProp<ViewStyle>;
+};
+
+type CellProps = {
+    // eslint-disable-next-line react/no-unused-prop-types
+    showTooltip: boolean;
+    isLargeScreenWidth: boolean;
+};
+
+type TaskCellProps = {
+    taskItem: TaskListItemType;
+} & CellProps;
+
+function DateCell({taskItem, showTooltip, isLargeScreenWidth}: TaskCellProps) {
```

Comment:
I think we have several definitions of DateCell. We could benefit from extracting this into a separate pure component that can be reused across all search ListItems